inputs/.gitattributes

```diff
@@ -0,0 +1,5 @@
+cholesky filter=lfs diff=lfs merge=lfs -text
+CMakeLists.txt filter=lfs diff=lfs merge=lfs -text
```

Comment:
this line should be removed

Reply:
maybe via `git lfs rm` or similar? I am not sure

Reply:
i mistakenly generated lfs attributes within inputs. I just recreated lfs from scratch.. something weird things went on. hope now it is resolved